README.md

```diff
@@ -204,6 +204,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [subdomainfinder.c99.nl](https://subdomainfinder.c99.nl/) - Scanner that scans an entire domain to find as many subdomains as possible
 - [AnubisDB](https://jonlu.ca/anubis/) - Subdomain enumeration and information gathering tool
 - [WhoisXMLAPI](https://www.whoisxmlapi.com/) - Domain & IP Data Intelligence for Greater Enterprise Security
+- [APIVoid](https://www.apivoid.com/) - Threat analysis centered on IP and Domain reputation, along with additional services
```

Comment:
imo this should go under the section `threat intelligence`

Reply:
@edoardottt no problem with me, but I'll be curious about WhoisXMLAPI, I use both of them, they are similar to me, maybe I should also me it?

Reply:
I'm don't know if I'm understanding correctly what you mean... if you mean that service should go to that category too I think not because it's a general purpose API, while this service is focused on threat analysis

Reply:
Oh, I mean WhoisXMLAPI is also multi-purpose threat intelligence, but very helpful for domain intelligence, just like APIVoid? Both of them has DNS lookup, Reverse DNS, Domain Reputation, Email Verification, SSL cert check, etc.

Reply:
ok, if you want to put that in ti is ok

Reply:
Maybe putting both of them into threat intelligence category is better, what do you think? I can do any of the versions, just want to provide some suggestion, but it's up to you :smile:

Reply:
we can put them both in TI